src/api/supabase/queries/plant_by_id.ts

```diff
@@ -0,0 +1,19 @@
+import { UUID } from 'crypto';
```

Comment:
this file is also duplicated in the `api/supabase/queries` route. let's delete the entire `src/` directory!